src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/meilisearch.css

```diff
@@ -92,23 +92,12 @@ div[data-ds-theme] .meilisearch-autocomplete {
 
 #search-bar-input {
   background-color: var(--pst-color-background);
-  border: 1px solid var(--pst-color-border);
-  border-radius: 0.25rem;
   color: var(--pst-color-text-base);
   font-size: var(--pst-font-size-icon);
   position: relative;
-  padding-left: 3rem;
-}
-
-.meilisearch-autocomplete::before {
-  content: "\f002";
-  font-family: "Font Awesome 6 Free";
-  position: absolute;
-  left: 8px;
-  top: 50%;
-  transform: translateY(-50%);
-  font-size: 1rem;
```

Comment:
So the autocomplete  is somewhere else now?

Reply:
yes, this was for icon using css, added that icon in js.